src/api/branch.go

```diff
@@ -67,7 +68,8 @@ func (a *StorageApi) DeleteBranch(branchId int) (*model.Job, error) {
 func (a *StorageApi) GetBranchRequest(branchId int) *client.Request {
 	branch := &model.Branch{}
 	return a.
-		NewRequest(resty.MethodGet, fmt.Sprintf("dev-branches/%d", branchId)).
+		NewRequest(resty.MethodGet, "dev-branches/{branchId}").
```

Comment:
Tu je bezpecnostna diera, .... respektive tu nie, lebo `%d` je integrer, ... al dalej, kde je `%s` string, a nad nim sa nevolalo `UrlEncode`, kniznica ma na to pekne API, tak som to prerobil aby to bolo bezpecne a nemohla sa modifikovat URL.